torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -21,14 +20,11 @@ def __init__(
         self,
         *,
         interpolation: InterpolationMode = InterpolationMode.NEAREST,
-        fill: Union[int, float, Sequence[int], Sequence[float]] = 0,
+        fill: Union[FillType, Dict[Type, FillType]] = 0,
```

Comment:
For BC:
```suggestion
        fill: Optional[Union[FillType, Dict[Type, FillType]]] = None,
```

same everywhere else?